test/jasmine/tests/mock_test.js

```diff
@@ -1815,8 +1815,8 @@ figs['mapbox_choropleth0'] = require('@mocks/mapbox_choropleth0');
 figs['mapbox_choropleth0-legend'] = require('@mocks/mapbox_choropleth0-legend');
 figs['mapbox_connectgaps'] = require('@mocks/mapbox_connectgaps');
 figs['mapbox_custom-style'] = require('@mocks/mapbox_custom-style');
-figs['mapbox_density-mulitple'] = require('@mocks/mapbox_density-mulitple');
-figs['mapbox_density-mulitple_legend'] = require('@mocks/mapbox_density-mulitple_legend');
+figs['mapbox_density-multiple'] = require('@mocks/mapbox_density-multiple');
+figs['mapbox_density-multiple_legend'] = require('@mocks/mapbox_density-multiple_legend');
```

Comment:
Could you please rename the mock and baseline files at following addresses?

test/image/mocks/mapbox_density-mulitple.json  
test/image/mocks/mapbox_density-mulitple_legend.json
test/image/baselines/mapbox_density-mulitple_legend.png
test/image/baselines/mapbox_density-mulitple.png